contracts/strategies/extensions/register/RecipientsExtension.sol

```diff
@@ -53,16 +53,6 @@ abstract contract RecipientsExtension is BaseStrategy, IRecipientsExtension, Err
     /// ========== Constructor =============
     /// ====================================
 
-    /// @notice Constructor to set the Allo contract
-    /// @param _allo Address of the Allo contract.
-    /// @param _strategyName Name of the strategy.
-    /// @param _reviewEachStatus true if custom review logic was added.
-    constructor(address _allo, string memory _strategyName, bool _reviewEachStatus)
-        BaseStrategy(_allo, _strategyName)
-    {
-        REVIEW_EACH_STATUS = _reviewEachStatus;
```

Comment:
^ @0xOneTony 

- Would it make sense to remove REVIEW_EACH_STATUS and always _processStatusRow ?
- Do the extensions need to have a constructor ?

Reply:
- Hmmm i think that if someone doesnt want to do some extra processing for each individual then its much cheaper to have `REVIEW_EACH_STATUS = false` and fill in the statuses, it gives a bit of flexibility.
- we used a constructor here because we have `REVIEW_EACH_STATUS ` immutable, usually extensions shouldn't need a constructor, they could use init function